jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessInstance.java

```diff
@@ -64,5 +65,9 @@
 	String getDeploymentId();
 	
 	void setDeploymentId(String deploymentId);
+	
+	Date getStartDate();
+	
+	void setStartDate(Date startDate);
```

Comment:
setter is certainly not needed as it should never be possible to set it other than process engine itself.